.ci/linux-steps.yaml

```
@@ -73,13 +77,17 @@ steps:
   displayName: 'CMake'
 
 # Build mlpack
-- script: cd build && make -j2
+- script: cd build && make -j2 
```

```suggestion
- script: cd build && make -j2
```
We will make the repository too large with one extra space :sob:

---

You need a file `.editorconfig` at the top to 'ever so gingerly' enfore policy across different editors different contributors may use.  

Duckdb shines and has [a rather extensive one](https://github.com/duckdb/duckdb/blob/main/.editorconfig), I tend to copy a [much smaller one](https://github.com/RcppCore/RcppArmadillo/blob/master/.editorconfig) from repo to repo.

I like `.editorconfig` because it is way less "in your face" than `clang-format` which I (personally) find obnoxious (with indentations choices made in many repos).  Then again, never ask me what I think of your and Conrad's style as we may not be on speaking terms thereafter :stuck_out_tongue_winking_eye:

---

I think the same about clang-format, it's a bit too much.  An .editorconfig might be nice but people like me who are developing using plain vi on 386s might have trouble with it :joy: (if someone contributed one I'd accept it, but personally I doubt I will make the time soon to create one)